.github/dependabot.yml

```diff
@@ -13,10 +13,10 @@ updates:
     schedule:
       interval: weekly
 
-    groups:
-      minor:
-        update-types:
-          - minor
-      patch:
-        update-types:
-          - patch
+groups:
```

Comment:
I'm not sure you can use `groups` as a top-level item like this. My impression was that it has to be nested under an item in `updates`. See how `groups` is mentioned under `updates` here: [Configuration options for the dependabot.yml file](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#configuration-options-for-the-dependabotyml-file).

In Comprehensive Rust, I simply duplicated the `groups` for each type of update.

Reply:
I see. I based my approach on [the example 1](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#example-1) in the official doc s but it might be just not a full example. I just amended the approach and duplicated the definitions.